firefox-ios/Client/Frontend/Browser/TopTabCell.swift

```diff
@@ -124,9 +124,9 @@ class TopTabCell: UICollectionViewCell, ThemeApplicable, LegacyTabTrayCell, Reus
 
         let isToolbarRefactorEnabled = featureFlags.isFeatureEnabled(.toolbarRefactor, checking: .buildOnly)
         let backgroundColor = isToolbarRefactorEnabled ? colors.actionTabActive : colors.layer2
-        selectedBackground.backgroundColor = backgroundColor
-        selectedBackground.layer.shadowColor = colors.shadowDefault.cgColor
-        selectedBackground.isHidden = false
+        cellBackground.backgroundColor = backgroundColor
```

Comment:
The background color for the selected tab should be `colors.layer2` for current toolbar and toolbar refactor.

Reply:
Ignore my previous comment. I showed design the screenshots with `layer2` and they were wrong: `actionTabActive` is correct when toolbar refactor is enabled.